pkgs/tools/networking/speedtest-rs/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, openssl
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "speedtest-rs";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "nelsonjchen";
+    repo = pname;
+    rev = "v${version}";
```

Sometime the automated update processes are running into issue if a tag/branch confusion is present.

```suggestion
    rev = "refs/tags/v${version}";
```